lib/import_export/topic_importer.rb

```diff
@@ -18,7 +18,7 @@ def perform
 
     def import_users
       @export_data[:users].each do |u|
-        existing = User.where(email: u[:email]).first
+        existing = User.with_email(email: u[:email]).first
```

Comment:
I'm not sure I understand: the scope is basically an alias for `.includes(:primary_email).where(user_emails: { email: ? }) }` so shouldn't it work the same as the original `.where` query?